packages/framework/src/Resources/views/Admin/Content/Watchdog/listGrid.html.twig

```diff
@@ -0,0 +1,28 @@
+{% extends '@ShopsysFramework/Admin/Grid/Grid.html.twig' %}
+
+{% block grid_value_cell_id_productName %}
+    {% if value %}
+        {{ value }}
+    {% else %}
+        {{ row.i.productCatnum }}
+        <span title="{{ 'The product is no longer in the catalog. This was the catalog number at the time of the watchdog.'|trans }}"
+              class="js-tooltip cursor-help in-icon in-icon--info"
+              data-toggle="tooltip"
+              data-placement="right"
+        >
+            {{ icon('info') }}
+        </span>
+    {% endif %}
+    {% if row.productId %}
```

Comment:
it does not seem like we can have the record without productId as there is a foreign key constraint... 
Maybe even the leftJoin in the repository method may be changed to join